project.py

```diff
@@ -136,8 +273,26 @@ def find_and_draw_edges(self, image, origin):
 
         img = origin.copy()
         cv.drawContours(img, [our_cnt], -1, (0, 255, 0), 3)
+        if(self.frame_counter % self.save_interval == 0):
+            self.write_image(our_cnt, img, mask)
+
+        self.frame_counter = (self.frame_counter + 1) % self.save_interval
         cv.imshow("img", img)
 
+    def preprocesing(self, image):
```

Comment:
@Foxiy This one looks pretty nice. Personally, I would add a parameter to make the `imshow` optional.